drafts/prometheus-alertmanager.adoc

```diff
@@ -0,0 +1,250 @@
+
+In this post, we'll set up Prometheus Alertmanager to fire alerts to a specified Slack channel to notify you when, for example, your app's heap usage is too high. We'll configure Prometheus with alerting rules to receive certain alerts from Open Liberty, then we'll configure Prometheus Alertmanager to pass those alerts to a Slack channel.
+
+Already familiar with Alertmanager? Check out link:https://github.com/jennifer-c/openliberty-alertmanager[a sample configuration for Open Liberty]. The repository also contains the configuration used as an example in this blog post, if you get stuck along the way.
+
+== What you'll need
+
+Before we start, make sure your app is configured with an Open Liberty server with the `mpMetrics-2.0` feature enabled.
+
+Download and extract both link:https://prometheus.io/download/#prometheus[Prometheus and Prometheus Alertmanager] to your laptop. We'll assume you know the basics of using Prometheus already.
+
+You also need a Slack channel to send alerts to, as well as a link:https://api.slack.com/messaging/webhooks[Slack webhook] for that channel.
+
+== Configure Prometheus to receive certain alerts from Open Liberty
+
+JENNIFER: Add a brief introduction here to say what the reader will achieve/learn from this section (configure Prometheus to work with OL and then set up alerting rules in Prometheus to xxxx).
+
+=== Set up Prometheus to communicate with Open Liberty
+JENNIFER: Add a sentence here saying what the reader will do here and to what end? where are these steps done (eg is the yaml file in Prometheus configuration?). eg "You need to configure Prometheus to communicate with the Open Liberty server in your app."
+
+. Edit the `prometheus.yml` file. Under the `scrape_configs` section, there is already a job configured for Prometheus itself. We need to add one for Open Liberty, which should look something like this:
++
+```
+- job_name: 'openliberty'
+
+    scheme: 'https'
+
+    tls_config:
+      insecure_skip_verify: true
+
+    static_configs:
+      - targets: ['localhost:9443']
+```
++
+. Run `./prometheus` (or `prometheus.exe` on Windows). When you see the message `Server is ready to receive web requests`, go to `http://localhost:9090` to view the Prometheus web UI.
+. To verify that your setup is working, click on `Targets` under `Status` in the navigation bar. You should see your `openliberty` target with the status `UP`.
+
+image::/img/blog/prometheusAM_ui_status.png[Prometheus Status on UI, align="left"]
+
+You're now ready to define which metrics you want Prometheus to collect.
+
+=== Creating Prometheus alerting rules
+
+Create some alerting rules in Prometheus to define which metrics will be received by Prometheus for passing on to Slack.
+
+. Make a file called `alert.yml`. (JENNIFER: Where?) Inside that file, create some alerting rules, for example:
```

Comment:
EDIT: Make a file called `alert.yml` anywhere - take note of the directory, as we will need it later.